### ISSUE TITLE
Feature request: specify event aggragation in the API

### ISSUE THREAD

**Issue by Jegp** [norse/aestream#32]
We are currently summing each events into a single polarity channel. 

Ideally, we would allow users to specify whether to aggregate the polarity, and how to do it. Specifically, we can describe three parameters

* Whether to aggregate polarity, or retain the information
  * Yes: `WxH` tensor
  * No: `WxHxP` tensor
* Count absolute or relative events
  * Unsigned: Sum absolute number of events
  * Signed: Subtract negative events from positive
* Limit count to 1 or count indefinitely
  * Threshold: binary value (bool data type?)
  * Indefinitely: 8-bit value (should suffice)

**Comment by cpehle** [comment]
I think the polarity is quite important and probably shouldn't be aggregated into one channel.

**Comment by Jegp** [comment]
I agree in principle, but there is a strong feature request to sum it here in Stockholm, which is why it's currently the default behavior. One of the reasons being memory/compute reductions, which is handy for our embedded systems.

I would suggest having both a summing and non-summing feature present, but default to maximum information (with polarity).

**Comment by cpehle** [comment]
I get that, it is more or less a requirement for efficiently processing the data. On the other hand demonstrating that you can handle the super high time resolution and sparse data should also be possible. I would be concerned that simply averaging over the polarity channels might loose quite a bit of temporal information.
